public/style.css

```diff
@@ -0,0 +1,247 @@
+@media only screen
+    and (min-device-width: 414px)
+    and (max-device-width: 736px) 
+    and (-webkit-min-device-pixel-ration: 3)
+    and (orientation: potrait) {
+        
+    }
+
+* {
+     padding: 0;
+     margin: 0;
+ }
+
+ body {
+     font-family: 'Josefin Slab', serif;
+     font-weight: bold;
+     word-wrap: break-word;
+     overflow-wrap: break-word;
+ }
+ 
+/********** main content areas **********/
+
+ #header {
+     display: flex;
+     justify-content: center;
+     align-items: center;
+     background: linear-gradient(rgb(129, 33, 129), rgb(196, 112, 196), rgb(129, 33, 129));     
+     width: 98vw;
+     height: 15vh;
+     margin: 0.5rem auto;
+     border-radius: 10px;
+     font-family: 'Signika', sans-serif;
+     font-size: 45px;
+ }
```

Comment:
If you add the rule ```cursor: pointer``` to your ```#header```, this will turn the mouse into a symbol that make it more clear that the header is a clickable element